### DISCUSSION TITLE
How to use "extract lora"?

### DISCUSSION THREAD

**Discussion by SAC020** [vladmandic/sdnext#3511] [General]
I cannot figure out how to use the new "extract lora" functionality.

I was assuming I can generate an image in T2I, then somehow extract a lora from that. However, when I press "refresh", nothing happens.

Is it because I am on Flux? Does it work only with SD1.5 and/or SDXL?

Log does not show any message, error or otherwise.

![image](https://github.com/user-attachments/assets/d2c1c7e5-0e52-4f33-833c-6bf09728c1af)

**Comment by vladmandic** [comment]
flux is coming. right now, it's sd15/21/xl.
